src/views/Home.jsx

```diff
@@ -8,12 +8,6 @@ export function Home({ handleClick, listToken, setListToken }) {
 	const [joinListName, setJoinListName] = useState('');
 	const [error, setError] = useState(false);
 
-	useEffect(() => {
```

Comment:
great work on solving the behavior with React, there were a lot of moving parts to figure out in the conditional render part!

Reply:
😅 truly! thank you, @lindseyindev!